vtm-playground/src/org/oscim/test/ExtrusionsTest.java

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2019 Gustl22
+ *
+ * This program is free software: you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY
+ * WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+ * PARTICULAR PURPOSE. See the GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with
+ * this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.oscim.test;
+
+import org.oscim.core.MapElement;
+import org.oscim.core.Tag;
+import org.oscim.gdx.GdxMapApp;
+import org.oscim.layers.TileGridLayer;
+import org.oscim.layers.tile.buildings.BuildingLayer;
+import org.oscim.layers.tile.buildings.S3DBLayer;
+import org.oscim.layers.tile.vector.VectorTileLayer;
+import org.oscim.test.tiling.source.TestTileSource;
+import org.oscim.theme.VtmThemes;
+
+public class ExtrusionsTest extends GdxMapApp {
+
+    enum GroundShape {
+        Rect, Hexagon, L, O,
```

Comment:
`L`, `O` ?

Reply:
Describes the shape of letter compared to the ground. If you know a better term, you're welcome.

Reply:
Didn't even understand what it is, thought Line and Outline.

Reply:
It describes the shape of the Letter: **L** and **O**. Search for [multi pitched](https://en.m.wikipedia.org/wiki/List_of_roof_shapes#Roof_shapes) roofs.

Reply:
For clarity better use full words:
```java
HEXAGON, RECTANGLE, SHAPE_L, SHAPE_O
```

Reply:
Done.